polymesh_schema.json

```
@@ -893,8 +893,8 @@
       }
     },
     "nft": {
-      "validateNFTTransfer": {
-        "description": "Verifies if and the sender and receiver are not the same, if both have valid balances, if the sender owns the nft, and if all compliance rules are being respected.",
+      "nft_transferReport": {
```

```suggestion
      "transferReport": {
```
I think the `nft_` prefix is added based on the module it is in.